### ISSUE TITLE
Sort the entries in Hacktoberfest_Inputt.xlsx

### ISSUE THREAD

**Issue by achoudh5** [achoudh5/Manipulating_Excel_Sheet#11]
- The entries in the **Hacktoberfest_Inputt.xlsx** file can be duplicate in certain rows, I would like to **sort the rows** based on either source or destination column (column D and F). if the same entries/cell values appear in rows then they should be one after another.

`For ex. `
Let's say in row 3->cell value is 1.2.3.4 and then in row 45 again the cell value is 1.2.3.4. In this situation, i would like to see row 45 below row 3 ie at row 4 and shift the rows from 4 down one position.

**Comment by Ishita-Mehta** [comment]
Hi! I would like to contribute for this issue. Hope it is okay if I use pandas library for the same. Have a good day!

P.S. Hacktoberfest released an update for maintainers hope this repo is complying to it https://hacktoberfest.digitalocean.com/hacktoberfest-update

**Comment by Ishita-Mehta** [comment]
Hi @achoudh5 Please add hacktoberfest to the topics of the repo as currently the PR is not being counted by digital ocean. These new rules are mentioned in the link in my previous comment. Thanks a lot!

**Comment by achoudh5** [comment]
Thank you @Ishita-Mehta for notifying, as far as I saw it says label issue with `hacktoberfest` and put `hacktoberfest-accepted` once PR gets approved right? Please suggest if there is anything I am missing, I want each one of contributors to get the credit of their work!

**Comment by achoudh5** [comment]
> Hi! I would like to contribute for this issue. Hope it is okay if I use pandas library for the same. Have a good day!
> 
> P.S. Hacktoberfest released an update for maintainers hope this repo is complying to it https://hacktoberfest.digitalocean.com/hacktoberfest-update

Yes, pandas is very welcome as well. Make sure you fetch and pull the repo as it's always work in progress :)

**Comment by Ishita-Mehta** [comment]
> Thank you @Ishita-Mehta for notifying, as far as I saw it says label issue with `hacktoberfest` and put `hacktoberfest-accepted` once PR gets approved right? Please suggest if there is anything I am missing, I want each one of contributors to get the credit of their work!

Hey, you need to add hacktoberfest as a topic for the repo as well. Below is what is mentioned in the update:

_'''We will honor all valid pull requests prior to this change, and as of October 3, 2020 at 12:00:00 UTC – and October 3 in all time zones – pull requests will only count toward earning a T-shirt or planting a tree if they are submitted in a **repository classified with the ‘hacktoberfest’ topic.** The pull requests will also need to be merged, approved by a maintainer, or labeled as ‘hacktoberfest-accepted’ in order to qualify. The deadline for completions, merging, labeling, and approving is November 1.

To put it in terms that folks will more easily understand:

PRs count if: 
**Submitted in a repo with the hacktoberfest topic AND 
during the month of October AND ( 
  The PR is merged OR 
  The PR is labelled as hacktoberfest-accepted by a maintainer OR 
  The PR has been approved** 
)'''_

**Comment by achoudh5** [comment]
> > Thank you @Ishita-Mehta for notifying, as far as I saw it says label issue with `hacktoberfest` and put `hacktoberfest-accepted` once PR gets approved right? Please suggest if there is anything I am missing, I want each one of contributors to get the credit of their work!
> 
> Hey, you need to add hacktoberfest as a topic for the repo as well. Below is what is mentioned in the update:
> 
> _'''We will honor all valid pull requests prior to this change, and as of October 3, 2020 at 12:00:00 UTC – and October 3 in all time zones – pull requests will only count toward earning a T-shirt or planting a tree if they are submitted in a **repository classified with the ‘hacktoberfest’ topic.** The pull requests will also need to be merged, approved by a maintainer, or labeled as ‘hacktoberfest-accepted’ in order to qualify. The deadline for completions, merging, labeling, and approving is November 1.
> 
> To put it in terms that folks will more easily understand:
> 
> PRs count if:
> **Submitted in a repo with the hacktoberfest topic AND during the month of October AND ( The PR is merged OR The PR is labelled as hacktoberfest-accepted by a maintainer OR The PR has been approved**
> )'''_

Just did and updated readme to thank you! Thank you so much :)
